src/common/components/feed/FeedCard.tsx

```diff
@@ -412,6 +479,30 @@ const OfferAction = (props: { nft: any }) => {
   );
 };
 
+export const ProfilePFPStack = ({ users }: { users: User[] }) => {
+  return (
+    <div className={`inline-flex items-center`}>
+      {users.slice(0, 3).map((user, i) => (
+        <div
+          key={user.address}
+          className={`${
+            i > 0 && `-ml-6`
+          } transition duration-100 ease-in hover:z-10 hover:scale-105`}
+        >
+          <ProfilePFP user={user} />
+        </div>
+      ))}
+      {users.length - 3 > 1 && (
```

Comment:
would be clearer as `users.length > 4` or, if you want to keep the `3` in there, then extract it to a constant and use it everywhere... here I would use it like `users.length > PFP_STACK_MORE_THRESHOLD + 1`

Also, should it be `users.length > 3`?

Reply:
good suggestion fixed